ext/standard/tests/array/array_intersect_assoc_variation5.phpt

```diff
@@ -67,8 +67,10 @@ foreach($arrays as $arr1) {
 
 echo "Done";
 ?>
---EXPECT--
+--EXPECTF--
 *** Testing array_intersect_assoc() : assoc array with diff keys to $arr1 argument ***
+
+Deprecated: Implicit conversion from non-compatible float 444.432 to int in %s on line %d
```

Comment:
This deprecation warning is thrown in an array literal for the test data. In such cases, I think it would be better to remove the element from the array / adjust the key, rather than adding the deprecation warning to the output. Otherwise there will be more work when this becomes an error.

The same also applies to some tests after this.